L1Trigger/TrackTrigger/test/CleanRelVal_cfg.py

```diff
@@ -0,0 +1,78 @@
+################################################################################################
+# Run bit-accurate TMTT L1 tracking emulation. 
+#
+# To run execute do
```

Comment:
Why is this job called CleanRelVal if it runs the TMTT tracking? Also, the comment inside it says it is called test_cfg.py.

Reply:
will be obsolete soon

Reply:
When will it be obsolete?

Reply:
already is.